### PR TITLE
This is the documentation for chatbot - how to debug Analytics Response

### PR DESCRIPTION
**Pull request by izairiabdlatif**

### CODE REVIEW

**Comment by smileEng** [comment]
Can we consider to host the image file within the repositort as well?
/image/<document-name-as-folder-name>/[your images file here]

and update the image tag within the markdown according to the latest hosted path

**Comment by izairiabdlatif** [comment]
> Can we consider to host the image file within the repositort as well?
> /image//[your images file here]
> 
> and update the image tag within the markdown according to the latest hosted path

Image paths updated to GIANT Documentation

File: connector/GoogleBigQuery/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Google BigQuery Connector Guide
+# Google BigQuery Connector GuideX
```
Comment (tchyong) [inline connector/GoogleBigQuery/README.md]: Why you change this file??

Comment (izairiabdlatif) [reply connector/GoogleBigQuery/README.md]: Accidentally added the X/changed the document when I referred to how the images paths are written. Ignore the changes, sorry!

**Pull request closed without merging**